core/commands/install_product_command.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+# Install product class
+
+# This class installs the product on selected node.
+class InstallProduct < BaseCommand
+  # This method is called whenever the command is executed
+  def execute
+    if @env.show_help
+      show_help
+      return SUCCESS_RESULT
+    end
+    return ARGUMENT_ERROR_RESULT unless init == SUCCESS_RESULT
+
+    if @mdbci_config.node_names.size != 1
+      @ui.error('Invalid node specified')
+      return ARGUMENT_ERROR_RESULT
+    end
+    machine = setup_ssh_key(@mdbci_config.node_names[0])
+    exit_code = install_product(machine)
+
+    exit_code
+  end
+
+  # Print brief instructions on how to use the command.
+  def show_help
+    info = <<-HELP
+ 'install_product'  Install a product onto the configuration node.
+ mdbci install_product config/node
```

Comment:
Install product command installs ANOTHER product onto the node, not the same one that was specified during the configuration!